wrappers/wasm/CMakeLists.txt

```diff
@@ -11,13 +11,11 @@ if (NOT CMAKE_BUILD_TYPE)
     set (CMAKE_BUILD_TYPE "MinSizeRel" CACHE STRING "Choose the type of build." FORCE)
 endif()
 
-add_definitions ("-s DISABLE_EXCEPTION_CATCHING=0")
-
 add_subdirectory (${CMAKE_CURRENT_SOURCE_DIR}/../../core ${CMAKE_BINARY_DIR}/ZXing)
 
 include_directories ("../../thirdparty/stb")
 
-set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} --bind -s ENVIRONMENT=web -s DISABLE_EXCEPTION_CATCHING=0 -s FILESYSTEM=0 -s MODULARIZE=1 -s EXPORT_NAME=ZXing")
+set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} --bind -s ENVIRONMENT=web -s DISABLE_EXCEPTION_CATCHING=0 -s FILESYSTEM=0 -s MODULARIZE=1 -s EXPORT_NAME=ZXing -s ALLOW_MEMORY_GROWTH")
```

Comment:
WASM throws OOM error after the above changes, I wonder if I did something wrong or did we just barely have enough memory?

Reply:
The error msg:

> Uncaught RuntimeError: abort(Cannot enlarge memory arrays to size 18034688 bytes (OOM). Either (1) compile with  -s INITIAL_MEMORY=X  with X higher than the current value 16777216, (2) compile with  -s ALLOW_MEMORY_GROWTH=1  which allows increasing the size at runtime, or (3) if you want malloc to return NULL (0) instead of this abort, compile with  -s ABORTING_MALLOC=0 ) at Error

Reply:
If you get an error after the change, why did you make the change in the first place?

Reply:
I mean that it errors after I expose `position` and adding `ALLOW_MEMORY_GROWTH` fixes it

Reply:
I see. That is indeed weird. You get the error even before you try to decode an image? If it happens during image decoding, then it certainly has to do with how large your image is and you might indeed just have hit the threshold just by adding those few lines? Unlikely things happen, I guess :).